File: 2. Loop Problems/Loop Problems in c/p24.c
// Write a C program to convert a decimal number into binary.

#include<stdio.h>
#include<math.h>


int main() {
    int n,remainder,position=1,summ=0;
    printf("Enter Valid Decimal Number: ");
    scanf("%d",&n);


    while (n!= 0) {
        remainder = n % 2;
        summ += remainder*position;
        n /= 2;
        position*=10;
    }
    printf("Binary: %d",summ);
    

}

File: 2. Loop Problems/Loop problems in python/p24.py
# // Write a python program to convert a decimal number into binary.

n = int(input("Enter a Valid decimal Number: "))


remainder = 0
position = 1
summ = 0


while n!=0:
    remainder = n % 2
    summ += remainder*position
    n = n // 2
    position*=10
    
print(f"Binary: {summ}")